src/core/Akka.Cluster.Tests/ClusterConfigSpec.cs

```diff
@@ -23,40 +25,49 @@ public class ClusterConfigSpec : AkkaSpec
-            Assert.Equal(TimeSpan.FromMilliseconds(33), settings.SchedulerTickDuration);
-            Assert.Equal(512, settings.SchedulerTicksPerWheel);
-            Assert.True(settings.MetricsEnabled);
-            //TODO:
-            //Assert.AreEqual(typeof(SigarMetricsCollector).FullName, settings.MetricsCollectorClass);
-            Assert.Equal(TimeSpan.FromSeconds(3), settings.MetricsGossipInterval);
-            Assert.Equal(TimeSpan.FromSeconds(12), settings.MetricsMovingAverageHalfLife);
-            Assert.Equal(false, settings.VerboseHeartbeatLogging);
+            settings.LogInfo.Should().BeTrue();
+
+            settings.SeedNodes.Should().BeEquivalentTo(ImmutableList.Create<Address>());
+            settings.SeedNodeTimeout.Should().Be(5.Seconds());
+            settings.RetryUnsuccessfulJoinAfter.Should().Be(10.Seconds());
+            settings.PeriodicTasksInitialDelay.Should().Be(1.Seconds());
+            settings.GossipInterval.Should().Be(1.Seconds());
+            settings.GossipTimeToLive.Should().Be(2.Seconds());
+            settings.HeartbeatInterval.Should().Be(1.Seconds());
+            settings.MonitoredByNrOfMembers.Should().Be(5);
+            settings.HeartbeatExpectedResponseAfter.Should().Be(1.Seconds());
+            settings.LeaderActionsInterval.Should().Be(1.Seconds());
+            settings.UnreachableNodesReaperInterval.Should().Be(1.Seconds());
+            settings.PublishStatsInterval.Should().NotHaveValue();
+            settings.AutoDownUnreachableAfter.Should().NotHaveValue();
+            settings.DownRemovalMargin.Should().Be(TimeSpan.Zero);
+            settings.MinNrOfMembers.Should().Be(1);
+            settings.MinNrOfMembersOfRole.Should().Equal(ImmutableDictionary<string, int>.Empty);
+            settings.Roles.Should().BeEquivalentTo(ImmutableHashSet<string>.Empty);
+            // settings.UseDispatcher.Should().Be(Dispatchers.DefaultDispatcherId);
+            settings.GossipDifferentViewProbability.Should().Be(0.8);
+            settings.ReduceGossipDifferentViewProbability.Should().Be(400);
+
+            Type.GetType(settings.FailureDetectorImplementationClass).Should().Be(typeof(PhiAccrualFailureDetector));
+            settings.FailureDetectorConfig.GetTimeSpan("heartbeat-interval").Should().Be(1.Seconds());
+            settings.FailureDetectorConfig.GetDouble("threshold").Should().Be(8.0d);
+            settings.FailureDetectorConfig.GetInt("max-sample-size").Should().Be(1000);
+            settings.FailureDetectorConfig.GetTimeSpan("min-std-deviation").Should().Be(100.Milliseconds());
+            settings.FailureDetectorConfig.GetTimeSpan("acceptable-heartbeat-pause").Should().Be(3.Seconds());
+            settings.FailureDetectorConfig.GetInt("monitored-by-nr-of-members").Should().Be(5);
+            settings.FailureDetectorConfig.GetTimeSpan("expected-response-after").Should().Be(1.Seconds());
+
+            // TODO remove metrics
```

Comment:
Before the 1.1 release we'll remove the Cluster.Metrics code and stick it into the Cluster.Metrics library, even if we don't release a nuget package for it yet. That way we don't have to worry about preserving these settings inside Akka.Cluster going forward.